study.md

```diff
@@ -31,7 +33,9 @@ In your own words explain "this".  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+T..his is a syntax that ALWAYS refers to and holds the value of an object. It is usually used inside a function or method and can also be used outside the function in the global scope.
```

Comment:
Be careful using the word ALWAYS here because `this` can be a quite tricky, as for now though that explanation is accurate.